libraries/AP_Proximity/AP_Proximity_AirSimSITL.cpp

```diff
@@ -34,15 +34,17 @@ void AP_Proximity_AirSimSITL::update(void)
 
     set_status(AP_Proximity::Status::Good);
 
-    memset(_distance_valid, 0, sizeof(_distance_valid));
+    for(uint8_t i=0; i < PROXIMITY_NUM_SECTORS; i++) {
```

Comment:
nitpick: add space after "for"

Reply:
we should ideally replace this with a single call to "AP_Proximity::mark_horizontal_sectors_invalid()" or ("mark_all_sectors_invalid" or "invalidate_all_sector_distances") which in turn calls a similarly named method on Boundary_3D.  Once we add support for more than one proximity sensors we will have problems with this method though.  Still, we can cross that bridge when we reach it.